dist/src/assembly/bin/zeneventserver-functions.sh

```diff
@@ -31,6 +33,8 @@ run() {
 }
 
 run_quiet() {
+    update_schema_in_background
```

Comment:
I thought ZEP was required to reach the DB. It will certainly be easier to just update the DB before starting ZEP...